HashMapSort/src/test/java/com/AndrieievYurii/ElementsDistributionTest.java

```diff
@@ -0,0 +1,117 @@
+package com.AndrieievYurii;
+
+import org.junit.jupiter.api.Test;
+
```

Comment:
You should separate only static imports from non-static imports with empty line. Don`t separate non-static from non-static